src/js/tech/ChromecastTech.js

```diff
@@ -181,20 +244,28 @@ ChromecastTech = {
       this._isMediaLoading = true;
       this._hasPlayedCurrentItem = false;
       castSession.loadMedia(request)
-         .then(function() {
-            if (!this._hasPlayedAnyItem) {
-               // `triggerReady` is required here to notify the Video.js player that the
-               // Tech has been initialized and is ready.
-               this.triggerReady();
-            }
-            this.trigger('loadstart');
-            this.trigger('loadeddata');
-            this.trigger('play');
-            this.trigger('playing');
-            this._hasPlayedAnyItem = true;
-            this._isMediaLoading = false;
-            this._getMediaSession().addUpdateListener(this._onMediaSessionStatusChanged.bind(this));
-         }.bind(this), this._triggerErrorEvent.bind(this));
+         .then(this.onLoadSessionSucces.bind(this), this._triggerErrorEvent.bind(this));
+   },
+
+   /**
+    * onLoadSessionSucces
+    */
+   onLoadSessionSucces: function() {
+      if (!this._hasPlayedAnyItem) {
+         // `triggerReady` is required here to notify the Video.js player that the
+         // Tech has been initialized and is ready.
+         this.triggerReady();
+      }
+
+      this.trigger('loadstart');
+      this.trigger('loadeddata');
+      this.trigger('play');
+      this.trigger('playing');
+      this.videojsPlayer.hasStarted(true);
+      this._hasPlayedAnyItem = true;
+      this._isMediaLoading = false;
+      window.setTimeout(this._onPlayerStateChanged.bind(this), 1000); // magic number 1 second to validate playstate b/c race conditions
```

Comment:
Not 100% sure, but should this timeout be cleaned up?

Reply:
Yes done